rasa/src/Rasa/Ext.hs

```diff
@@ -153,7 +154,6 @@ module Rasa.Ext
   , asCoord
   , clampCoord
   , clampRange
-  , range
```

Comment:
We should do like you did with text and keep this in (but just as a getter); getting ranges of text is probably going to be a common operation.